pages/api/book/event.ts

```diff
@@ -350,8 +357,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         selectedCalendars
       );
 
-      const videoBusyTimes = await getBusyVideoTimes(credentials);
+      const videoBusyTimes = (await getBusyVideoTimes(credentials)).filter((time) => time);
       calendarBusyTimes.push(...videoBusyTimes);
+      console.log("calendarBusyTimes==>>>", calendarBusyTimes);
```

Comment:
This is throwing an error when trying to book a regular Zoom booking with Daily disabled (DAILY_API_KEY is undefined)

![image](https://user-images.githubusercontent.com/3504472/136072549-1c3886fd-23b1-47a9-9ce3-c5de53966444.png)

Reply:
Actually never mind. It seems that's a separate issue with Zoom. You can disregard this. All good now.